api/kafka_endpoints.py

```
@@ -0,0 +1,382 @@
+async def get_kafka_status() -> Dict[str, Any]:
+    """Kafka Consumer 상태 조회"""
+    try:
+        status = kafka_manager.get_status()
+        return {
+            "status": "success",
+            "data": status,
+            "message": "Kafka Consumer 상태 조회 성공"
+        }
+    except Exception as e:
+        logger.error(f"Kafka 상태 조회 실패: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"상태 조회 실패: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)